### ISSUE TITLE
Add setting to flip video Horizontally in player without flipping controls

### ISSUE THREAD

**Issue by rvddekock** [apivideo/api.video-player-sdk#44]
**Is your feature request related to a problem? Please describe.**
When we record a person we flip the camera so that it will look to the user that they are looking into a mirror. but when we switch to the recorded video it flips back the other way around. 

**Describe the solution you'd like**
So we want a setting we can give the player sdk to mirror/flip the video horizontally to play the video as if the user is watching them self in the mirror.

**Comment by olivierapivideo** [comment]
Hi Rudolph,

With the last released version (v1.2.20) you can do:
```javascript
player.setVideoStyleTransform("rotateY(180deg)"); // Apply a 180deg rotation around the Y axis (mirroring)
``` 
Best,

Olivier

**Comment by tpak** [comment]
@olivierapivideo thanks, that appears to do the trick!
